### ISSUE TITLE
[a11y]: ContentSwitcher not working as expected when used within a CreateTearsheetStep

### ISSUE THREAD

**Issue by mariyageorge01** [carbon-design-system/ibm-products#7684]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

MacOS

### Package version

2.64.0

### Automated testing tool and ruleset

Keyboard navigation

### Description

The carbon component ContentSwitcher is not working as expected when it is used within a CreateTearsheetStep.

### WCAG 2.1 Violation

2.1.1 Keyboard

### Reproduction/example

https://stackblitz.com/edit/github-p9rgggfv?file=src%2FExample%2FExample.jsx

### Steps to reproduce

Using tab keys and arrow keys to navigate ContentSwitcher tabs are not working as expected. It should work as per https://carbondesignsystem.com/components/content-switcher/accessibility/#keyboard-interactions

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
